pysonde/sounding.py

```diff
@@ -388,10 +388,16 @@ def set_launchtime(self):
 
     def export(self, output_fmt, cfg):
         """
-        Saves sounding to disk
+        Saves sounding to disk with correctly formatted filename.
+
+        - Uses platform from `cfg.main.get("platform")` instead of extracting from filename.
```

Comment:
As we only document the current version, we do not need to reference past implementations here:

```suggestion
        - Uses platform from `cfg.main.get("platform")`
```